### ISSUE TITLE
bug: Search by content often causes unresponsive site (page freeze)

### ISSUE THREAD

**Issue by wasnertobias** [windmill-labs/windmill#3885]
### Describe the bug

Don't know what it causes it. But often the whole page freezes.

### To reproduce

Not sure. Just search for stuff.

### Expected behavior

No freezing...

**Comment by wasnertobias** [comment]
Example string which I just searched for: "can-not_tell_you-theexactcontent"

**Comment by rubenfiszel** [comment]
It likely depends more on the content of the workspace than the string itself. Just to confirm, you are on EE (and thus loading the entire workspace). How many scripts, flows, apps do you have in there?

**Comment by wasnertobias** [comment]
Yes, on EE. 306 scripts/flows atm. No apps so far.

**Comment by rubenfiszel** [comment]
The implementation is very naive, there are likely low hanging fruits we can improve easily. Thanks for the report

**Comment by wasnertobias** [comment]
> "It likely depends more on the content of the workspace than the string itself."

I tend to disagree; with the search string provided above, I am sure you will also be able to freeze your content search.
